datumaro/plugins/vott_json_format.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import json
+import os.path as osp
+
+from datumaro.components.annotation import AnnotationType, Bbox, LabelCategories
+from datumaro.components.extractor import DatasetItem, Importer, SourceExtractor
+from datumaro.components.format_detection import FormatDetectionContext
+from datumaro.components.media import Image
+from datumaro.util.meta_file_util import has_meta_file, parse_meta_file
+
+
+class VottJsonPath:
+    ANNO_FILE_SUFFIX = '-export.json'
+
+class VottJsonExtractor(SourceExtractor):
+    def __init__(self, path):
+        if not osp.isfile(path):
+            raise FileNotFoundError("Can't read annotation file '%s'" % path)
+
+        super().__init__(subset=osp.splitext(osp.basename(path))[0].split('-')[0])
+
+        self._categories = { AnnotationType.label: LabelCategories() }
+        if has_meta_file(path):
+            self._categories = { AnnotationType.label: LabelCategories().
+                from_iterable(list(parse_meta_file(path).keys())) }
+
+        self._items = list(self._load_items(path).values())
+
+    def _load_items(self, path):
+        items = {}
+
+        with open(path) as f:
+            anno_dict = json.load(f)
+
+        label_categories = self._categories[AnnotationType.label]
+        tags = anno_dict.get('tags', [])
+        for label in tags:
+            label_name = label.get('name')
+            if label_name:
+                label_categories.add(label_name)
```

Comment:
Won't this conflict with categories that were added from the meta file?

Reply:
I think not. We're just adding a class if it doesn't exist yet.